### PR TITLE
Improve person states handling

### PR DESCRIPTION
**Pull request by ioigoume**

### CODE REVIEW

File: dictionaries/noty.definition.json [hunk 1]
```diff
@@ -56,6 +56,12 @@
     "user_suspended_title": {
         "en": "Identity suspended"
     },
+    "user_error": {
```
Comment (NicolasLiampotis) [inline dictionaries/noty.definition.json:59]: There is a problem with your identifier (%ORGID%). Please get in touch with our support team at <a href="mailto:support@rciam.example.org">support@rciam.example.org</a> and include the error message shown above. We apologise for any inconvenience caused and will help you resolve this issue as quickly as possible.

File: dictionaries/noty.definition.json [hunk 2]
```diff
@@ -56,6 +56,12 @@
...
+        "en": "Your identity (%ORGID%) is in error state. Please contact our <a href=\"mailto:support@rciam.example.org\">support</a> and make sure to include the error message above"
+    },
+    "user_error_title": {
+        "en": "Identity Error"
```
Comment (NicolasLiampotis) [inline dictionaries/noty.definition.json:63]: Sorry, we were not able to log you in

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
See suggestions for updated error title & message

File: dictionaries/noty.definition.json (hunk 1 above)
Comment (ioigoume) [reply dictionaries/noty.definition.json:59]: Done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**